### ISSUE TITLE
Microtonality support

### ISSUE THREAD

**Issue by suhr** [calf-studio-gear/calf#86]
See [MIDI Tuning Standard](http://www.microtonal-synthesis.com/MIDItuning.html) and [Scala file format](http://www.huygens-fokker.org/scala/scl_format.html).

Additional info: [31-TET](http://31et.com/) is [fun](http://tinyurl.com/zwf4jm5).

**Comment by SeanArchibald** [comment]
Agreed, microtonal support would make the Calf instruments really appealing for me. I don't write non-microtonal music at all these days, and I'm struggling to find VST instruments that support microtonal scales as I move over to a Linux-based studio.

suhr rightly mentioned MTS and Scala formats. There is also the AnaMark TUN format, which stores the tuning in a readable text file with the extension .tun. There is an open-source implementation of this written in C++ which could be brought into this project in order to provide the missing functionality. Check http://www.mark-henning.de/am_downloads_eng.php for more info (scroll down near the bottom for a linux to the source code).

Only one kind of tuning format has to be implemented, so there is at least one way to allow us musicians to use microtonal scales!

I'm always available to consult about this, I would be more than happy to fill any gaps in understanding about microtonality.

**Comment by SeanArchibald** [comment]
One more option: re-use the microtonal functions that are present in ZynAddSubFX - they have already done good work in supporting Scala .scl/.kbm format :)
